cypress/tests/integration/captures/[captureid].cy.js

```diff
@@ -0,0 +1,63 @@
+import moment from 'moment';
+import capture from '../../../fixtures/capture.json';
+import country from '../../../fixtures/country.json';
+import grower from '../../../fixtures/grower.json';
+import { prepareNocks, clearNocks } from '../nockRoutes';
+
+beforeEach(() => {
+  clearNocks();
+});
+
+describe('Capture page', () => {
```

Comment:
@dadiorchen cypress integration test for the capture page has been added and passed. could you kindly take a look? i have requested another review from you. thanks!